mosec/log.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2022 MOSEC Authors
```

Comment:
```suggestion
# Copyright 2023 MOSEC Authors
```

Reply:
LOL, do we need to update all the files?

Reply:
> followed by the year of the first publication of the work
https://en.wikipedia.org/wiki/Copyright

My view is that new files can use the current year?

Reply:
SGTM.